### PR TITLE
More Bugfixes

### LINKED ISSUE #18227: [Bug]: Orion Express outpost (2) spawns with destinationless packages.
### Checks

- [X] I have searched the bug with a few keywords, and I confirm this bug was not yet reported.

### Location

Away Site

### Describe the issue

https://github.com/Aurorastation/Aurora.3/pull/18135 caused the OX delivery points to be changed (for the better), and alongside them were added packages that do not properly set a destination nor credit value on roundstart, leaving you with packages that go nowhere.

### How to reproduce

1. Generate an Exoplanet with a delivery point
2. Observe package (without destination)

### Round ID

crM-dqqG

### Staff ckey

_No response_

### PR DESCRIPTION
**Pull request by RustingWithYou**
Fixes #18227 and #18172
Fixes the Tempest SMG not using its wielded icon state.
Fixes exoplanet plants looking for the wrong number of growth stages and constantly runtiming.

### CODE REVIEW

**Comment by RustingWithYou** [comment]
!bugfix

**Comment by RustingWithYou** [comment]
!review

File: code/modules/mob/living/simple_animal/friendly/adhomai.dm [hunk 1]
```diff
@@ -283,11 +283,6 @@
 	meat_amount = 4
 	faction = "Adhomai"
 	vehicle_version = /obj/vehicle/animal/climber
-	natural_armor = list(
-		melee = ARMOR_MELEE_MEDIUM,
-		bullet = ARMOR_BALLISTIC_MINOR,
-		bomb = ARMOR_BOMB_MINOR
-	)
```
Comment (FluffyGhoster) [inline code/modules/mob/living/simple_animal/friendly/adhomai.dm:290]: I am not sure removing the armor is a proper solution for the runtime

Comment (RustingWithYou) [reply code/modules/mob/living/simple_animal/friendly/adhomai.dm]: Removed this as the issue was solved properly by #18856

**Pull request merged**